fjord/analytics/templates/analytics/analyzer/hb_surveys.html

```diff
@@ -1,74 +1,91 @@
-            Older
-          </a>
-        {% else %}
-          Older
-        {% endif %}
-        | Page {{ surveys.number }} of {{ surveys.paginator.num_pages }} |
-        {% if prev_page %}
-          <a class="newer" href="{{ request.get_full_path()|urlparams(page=surveys.next_page_number) }}">
-            Newer
-          </a>
-        {% else %}
-          Newer
-        {% endif %}
-      </div>
+  <div class="pager">
+    {% if next_page %}
+    <a class="older" href="{{ request.get_full_path()|urlparams(page=surveys.previous_page_number) }}">
+      Older
+    </a>
+    {% else %}
+    Older
+    {% endif %}
+    | Page {{ surveys.number }} of {{ surveys.paginator.num_pages }} |
+    {% if prev_page %}
+    <a class="newer" href="{{ request.get_full_path()|urlparams(page=surveys.next_page_number) }}">
+      Newer
+    </a>
+    {% else %}
+    Newer
+    {% endif %}
+  </div>
 
-      <table class="summarytable">
-        <tr>
-          <th>name</th>
-          <th>description</th>
-          <th>enabled?</th>
-          <th>created</th>
-        </tr>
+  <table class="summarytable">
+    <tr>
+      <th>name</th>
+      <th>description</th>
+      <th>enabled?</th>
+      <th>created</th>
+    </tr>
 
-        {% for sur in surveys %}
-          <tr>
-            <td><pre>{{ sur.name }}</pre></td>
-            <td>{{ sur.description }}</td>
-            <td>{{ sur.enabled }}</td>
-            <td>{{ sur.created }}</td>
-          </tr>
-        {% endfor %}
-      </table>
+    {% for sur in surveys %}
+    <tr>
+      <td><pre>{{ sur.name }}</pre></td>
+      <td>{{ sur.description }}</td>
+      <td>{{ sur.enabled }}</td>
+      <td>{{ sur.created }}</td>
+    </tr>
+    {% endfor %}
+  </table>
 
-      <div class="pager">
-        {% if next_page %}
-          <a class="older" href="{{ request.get_full_path()|urlparams(page=surveys.previous_page_number) }}">
-            Older
-          </a>
-        {% else %}
-          Older
-        {% endif %}
-        | Page {{ surveys.number }} of {{ surveys.paginator.num_pages }} |
-        {% if prev_page %}
-          <a class="newer" href="{{ request.get_full_path()|urlparams(page=surveys.next_page_number) }}">
-            Newer
-          </a>
-        {% else %}
-          Newer
-        {% endif %}
-      </div>
-    </div>
+  <div class="pager">
+    {% if next_page %}
+    <a class="older" href="{{ request.get_full_path()|urlparams(page=surveys.previous_page_number) }}">
+      Older
+    </a>
+    {% else %}
+    Older
+    {% endif %}
+    | Page {{ surveys.number }} of {{ surveys.paginator.num_pages }} |
+    {% if prev_page %}
+    <a class="newer" href="{{ request.get_full_path()|urlparams(page=surveys.next_page_number) }}">
+      Newer
+    </a>
+    {% else %}
+    Newer
+    {% endif %}
   </div>
+  <h2>Create survey</h2>
+  <form action='' method='POST'>
+    {{ csrf() }}
+    {% for field in form %}
+    <p>
```

Comment:
Sure! But we care less about the output HTML than we do about readability of the template code and maintenance.